specification/cognitiveservices/OpenAI.Inference/models/audio/common.tsp

```diff
@@ -13,3 +15,9 @@ enum AudioTaskLabel {
   @doc("Accompanying response data resulted from an audio translation task.")
   translate: "translate",
 }
+
+alias AudioTextResponse = {
+  @doc("The text response data.")
```

Comment:
I would phrase this like we did for the request header trait:

@doc("The content type for the response. Always text/plain for this operation.")

Reply:
This one describes the response text / string. So I'd assume we say e.g. `The Response data of content type "text/plain".`?